index.html

```diff
@@ -0,0 +1,275 @@
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-4">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+
+            <div class="view-project-4">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-me-page">
+
+            <div class="about-me-title">
+                <h3> About <br> Myself</h3>
+            </div>
+
+            <div class="about-me-p">
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="image/angellist.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="image/Linkedinicon.png" alt="" /></a></li>
+                </ul>
+            </div>
+            </div>
+            <div class="resume">
+                <button type="button"> Get my resume </button>
+            </div>
+            <div class="skill">
+              <ul>
+                <li>
+                  <ul>
+                    <li class="lang">
+                      <p class="">Languages</p>
+                    </li>
+                  </ul>
+                  <ul class="my-lang">
+                    <li class="js">
+                      <img src="image/js.png" alt="" />
+                      <p>JAVASCRIPT</p>
+                    </li>
+                    <li class="html">
+                      <img src="image/html.png" alt="" />
+                      <p>HTML</p>
+                    </li>
+                    <li class="css">
+                      <img src="image/css.png" alt="" />
+                      <p>CSS</p>
+                    </li>
+                  </ul>
+                </li>
+                <li class="frameworks">
+                  <p class="">Frameworks</p>
+                </li>
+                <li class="skills">
+                  <h2> Skills </h2>
+                </li>
+              </ul>
+            </div>
```

Comment:
- [ ] If you see the spacing between `Framework` and `Skills` then there is a lot so kindly fix it as review by previous reviewer.
- [ ] kindly fix the issue on `skills` as well.
## figma design
![his design](https://user-images.githubusercontent.com/81727988/164741526-3067191f-09e7-4f95-9a9c-03bfdcf433f2.png)

## your
![hisimage](https://user-images.githubusercontent.com/81727988/164741866-cff1e3b3-035b-4a11-87e0-65989705dc6d.png)

Reply:
sorry i forgot to commit the changes

Reply:
both issues were fixed